### PR TITLE
[NCL-7080] Refactor CLI to read logs of completed builds from Bifrost

### PR DESCRIPTION
**Pull request by jomrazek**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Review by thescouser89 — APPROVED** [review]

File: pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java [hunk 1]
```diff
@@ -31,9 +37,29 @@ public BifrostClient(URI baseUrl) {
...
+    }
+
+    public void writeLog(String id, boolean follow, Consumer<String> onLine, LogType logType) throws IOException {
+
+        String query;
+
+        switch (logType) {
+            case BUILD:
+                query = "direction=ASC" + "&matchFilters=mdc.processContext.keyword:build-" + id
+                        + "&prefixFilters=loggerName.keyword:org.jboss.pnc._userlog_.build-log";
```
Comment (michalovjan) [inline pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java:53]:
```
MessageFormat.format("direction=ASC&matchFilters=mdc.processContext:build-{0},loggerName:org.jboss.pnc._userlog_.build-log",id)
```
Cleaner code and maybe more optimized by moving loggerName to matchFilter. 

For BUILD and ALIGNMENT, to increase performance, add smthing alongside `&batchSize=5000` and `&batchDelay=500`. 

Plus, there are additional formats to log lines. You can configure with `&format=XXXX`. Formats are TIMESTAMP (timestamp + message), PLAIN (just message), LEVEL (logLevel + message), DEFAULT (timestamp + loggerName + message). 

Are you using `/text` or `/` endpoint?

File: pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java [hunk 2]
```diff
@@ -31,9 +37,29 @@ public BifrostClient(URI baseUrl) {
...
+                break;
+            case ALIGNMENT:
```
Comment (michalovjan) [inline pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java:55]:
```
MessageFormat.format("direction=ASC&matchFilters=mdc.processContext:build-{0},loggerName:org.jboss.pnc._userlog_.alignment-log",id)
```
Cleaner code and maybe more optimized by moving loggerName to matchFilter. 

For BUILD and ALIGNMENT, to increase performance, add smthing alongside `&batchSize=5000` and `&batchDelay=500`. 

Plus, there are additional formats to log lines. You can configure with `&format=XXXX`. Formats are TIMESTAMP (timestamp + message), PLAIN (just message), LEVEL (logLevel + message), DEFAULT (timestamp + loggerName + message). 

Are you using `/text` or `/` endpoint?

File: pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java [hunk 3]
```diff
@@ -31,9 +37,29 @@ public BifrostClient(URI baseUrl) {
...
+                query = "direction=ASC" + "&matchFilters=mdc.processContext.keyword:build-" + id
+                        + "&prefixFilters=loggerName.keyword:org.jboss.pnc._userlog_.alignment-log";
+                break;
+            default:
+                query = "direction=ASC" + "&matchFilters=mdc.processContext.keyword:build-" + id
```
Comment (michalovjan) [inline pnc/src/main/java/org/jboss/pnc/bacon/pnc/client/BifrostClient.java:60]:
```
MessageFormat.format("direction=ASC&matchFilters=mdc.processContext:build-{0}&prefixFilters=loggerName:org.jboss.pnc._userlog_",id)
```
For BUILD and ALIGNMENT, to increase performance, add smthing alongside `&batchSize=300` and `&batchDelay=500`. 

Plus, there are additional formats to log lines. You can configure with `&format=XXXX`. Formats are TIMESTAMP (timestamp + message), PLAIN (just message), LEVEL (logLevel + message), DEFAULT (timestamp + loggerName + message). 

Are you using `/text` or `/` endpoint?

**Review by michalovjan — COMMENTED** [review]
HI!

**Pull request merged**